### PR TITLE
Update to 12.81

### PR DESCRIPTION
**Pull request by Zbizu**
## New features
- client 12.71/12.72
- client 12.81
- two new official OTBs (3.59/client 12.72, 3.60/client 12.81), developed in cooperation with (and approval of) otland staff, backwards compatible with previous EPuncker's OTBs
- colourable mounts
- podium support (can be enabled two ways: item group `14` (`0x0E`) in **items.otb** or `<attribute key="type" value="podium" />` in **items.xml**)
obraz

## Technical changes
- invisible wall 1548 now displayed as cyan square for better visibility:
obraz
- slight cosmetic changes to properties dialog
- merged two functions responsible for drawing items into one (only one parameter was different so there was no need to have two large, almost identical functions)

## Possible compatibility with
- https://github.com/Zbizu/forgottenserver (full compatibility)
- https://github.com/otland/forgottenserver (requires https://github.com/otland/forgottenserver/pull/3858/files)
- https://github.com/opentibiabr/canary (requires https://github.com/opentibiabr/canary/pull/167, https://github.com/opentibiabr/canary/pull/161 << READ DESCRIPTION IF YOU WANT TO USE IT ON CANARY!)

## Format specification
legend:
1 byte = u8
2 bytes = u16

structure:
- 1 byte header: OTBM_ATTR_PODIUMOUTFIT (40 dec, 0x28 hex)
- 1 byte flags (showPlatform, showOutfit, showMount)
- 1 byte direction
- 2 bytes looktype
- 4x 1 byte outfit colors
- 1 byte addons
- 2 bytes mount looktype
- 4x 1 byte mount colors

## Notes
- As this is a new feature, any maps containing podiums may fail to load in outdated server engines unless they get patched with podium parsing (see https://github.com/opentibiabr/canary/pull/161/files for `propstream.read` example).

### CODE REVIEW

File: data/clients.xml [hunk 1]
```diff
@@ -284,6 +285,12 @@
             <extensions from="8.20 - 8.31" to="10.100"/>
             <data format="10.57" dat="0x44BA" spr="0x585138E5"/>
 		</client>
+		
+		<client name="12.71 - 12.72" otb="12.71" visible="true" data_directory="1271">
+            <otbm version="3"/>
+            <extensions from="8.20 - 8.31" to="12.71 - 12.72"/>
```
Comment (gpedro) [inline data/clients.xml:291]: broken indent

File: source/definitions.h [hunk 2]
```diff
@@ -50,7 +50,7 @@
 #endif
 // OS
 
-#define OTGZ_SUPPORT 1
+//#define OTGZ_SUPPORT 1
```
Comment (gpedro) [inline source/definitions.h:53]: any specific reason for disabling otgz support?

Comment (Zbizu) [reply source/definitions.h:53]: I ran into problems with linker in visual studio 2019 (the error is "unrecognized external symbol"):
obraz

I can reenable it for pull request purposes though

Comment (Zbizu) [reply source/definitions.h]: nvm, was bad configuration in my visual studio. I pushed a commit reverting that.

**Comment by Zbizu** [comment]
Added two small features:

- remove all invalid item ids from the map (maps with invalid items visible as red squares fail to load on the servers, this feature removes all of them)
obraz

- features such as "find by id", "remove by id" and "replace one item with another" now allow forcing server ID as another way to combat red squares
obraz

**Comment by omarcopires** [comment]
`remove all invalid item` this feature already exists and is called cleanup

File: data/clients.xml (hunk 1 above)
Comment (Zbizu) [reply data/clients.xml:292]: it's following the pattern of the versions above

**Comment by Zbizu** [comment]
I should've pushed it from a branch other than master. I don't have time to edit the code or split it to separate PRs.
Anyway, if someone is interested in doing that, I do consent using my edits.

**Pull request closed without merging**